### DISCUSSION TITLE
Suggestions question

### DISCUSSION THREAD

**Discussion by VictoryTek** [cmintey/wishlist#154] [Q&A] [answered]
I have Suggestions with approval required turned on, and have tried other approval modes. Maybe I am just missing something, but how do you suggest something for someone else. When I click on someone else's list I don't get a + to add anything. Any help would be appreciated. Thanks

**Comment by cmintey** [comment, marked as answer]
Hi, make sure you have the setting enabled (checkbox) for the group as well. Go to Admin > Groups > Select your group > Settings

**Reply by VictoryTek** [reply]
That was it, thanks so much.
